js/config.js

```diff
@@ -1,197 +1,128 @@
+                primary_btn: {
+                    text: 'Accept all',
+                    role: 'accept_all'              // 'accept_selected' or 'accept_all'
+                },
+                secondary_btn: {
+                    text: 'Reject all',
+                    role: 'accept_necessary'        // 'settings' or 'accept_necessary'
+                }
             },
-        },
-        {
-            name : 'tessitura',
-            title : 'Tessitura',
-            default: true,
-            purposes : ['functionality', 'security'],
-            callback : function(consent, app){
-                // This is an example callback function.
-                console.log("User consent for app "+app.name+": consent="+consent)
-            },
-        },
-        {
-            name : 'cloudflare',
-            title : 'Cloudflare',
-            default: true,
-            purposes : ['functionality', 'security'],
-            callback : function(consent, app){
-                // This is an example callback function.
-                console.log("User consent for app "+app.name+": consent="+consent)
-            },
-        },
-        {
-            name : 'queueit',
-            title : 'Queue-It',
-            default: true,
-            purposes : ['functionality', 'security'],
-            cookies : [/^Queue/i],
-            callback : function(consent, app){
-                // This is an example callback function.
-                console.log("User consent for app "+app.name+": consent="+consent)
-            },
-        },
-        {
-            name : 'bugherd',
-            title : 'Bugherd',
-            default: true,
-            purposes : ['analytics'],
-            callback : function(consent, app){
-                // This is an example callback function.
-                console.log("User consent for app "+app.name+": consent="+consent)
-            },
-        },
-    ],
-}
+            settings_modal: {
+                title: logo,
+                save_settings_btn: 'Save my choices',
+                accept_all_btn: 'Accept all',
+                reject_all_btn: 'Reject all',
+                close_btn_label: 'Close',
+                cookie_table_headers: [
+                    {col1: 'Name'},
+                    {col2: 'Domain'},
+                    {col3: 'Expiration'},
+                    {col4: 'Description'}
+                ],
+                blocks: [
+                    {
+                        title: 'Cookie usage and the Fitzwilliam website',
+                        description: 'The Fitzwilliam Museum uses cookies to ensure the basic functionalities of the website and to enhance your online experience. You can choose for each category to opt-in/out whenever you want. For more details relative to cookies and other sensitive data, please read the full <a href="#" class="cc-link">privacy policy</a>.'
```

Comment:
The privacy policy needs linking correctly.

I'm guessing it should be linked here:

https://fitzmuseum.cam.ac.uk/about-us/privacy-and-cookies

This then directs users to the University privacy policy here:

https://www.cam.ac.uk/about-this-site/privacy-policy

Linking directly to the cam.ac.uk page would probably be confusing for users.